<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple-2DPlot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="8">
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 243);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>f(x):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>sin(10*x)*exp(-x)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>x from:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>to</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>2*pi</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>171</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Plot!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>197</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
